File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/nuro/NuroOUT0101U02GetJohapInfo.java
package nta.med.data.model.ihis.nuro;

import java.io.Serializable;

public class NuroOUT0101U02GetJohapInfo implements Serializable{
	 private String johapName;
	 private String johapGubun;
	public NuroOUT0101U02GetJohapInfo(String johapName, String johapGubun) {
		super();
		this.johapName = johapName;
		this.johapGubun = johapGubun;
	}
	public String getJohapName() {
		return johapName;
	}
	public void setJohapName(String johapName) {
		this.johapName = johapName;
	}
	public String getJohapGubun() {
		return johapGubun;
	}
	public void setJohapGubun(String johapGubun) {
		this.johapGubun = johapGubun;
	}
	
	
	 
	 
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Nuro/NuroOUT0101U02GetJohapInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Nuro
{
    [Serializable ]
	public class NuroOUT0101U02GetJohapInfo
	{
		private String _johapName;
		private String _johapGubun;

		public String JohapName
		{
			get { return this._johapName; }
			set { this._johapName = value; }
		}

		public String JohapGubun
		{
			get { return this._johapGubun; }
			set { this._johapGubun = value; }
		}

		public NuroOUT0101U02GetJohapInfo() { }

		public NuroOUT0101U02GetJohapInfo(String johapName, String johapGubun)
		{
			this._johapName = johapName;
			this._johapGubun = johapGubun;
		}

	}
}